transformations/casual2formal/README.md

```diff
@@ -0,0 +1,15 @@
+# Casual to Formal 🦎 → 🐍
+This transformation transfers text style from casual (informal) to formal.
+
+Author Name: Rishabh Gupta (rishabh19089@iiitd.ac.in)
+
+## What type of a transformation is this?
+This transformation acts like a paraphraser by performing style transfer. It changes the tone of the text from informal to formal.
+Example: What you upto? --> What are you currently doing? 
+
+## What tasks does it intend to benefit?
+This perturbation would benefit all tasks which have a sentence/paragraph/document as input like text classification, 
+text generation, etc. 
+
+## What are the limitations of this transformation?
+The transformation's outputs may lose some information in case of longer sequences and may not be exact.
```

Comment:
Maybe you could add some of the bad generations too in the test file?

Reply:
Added negative sample